docs/devices/BRT-100-TRV.md

```diff
@@ -31,6 +31,16 @@ While pairing, keep the valve close to the coordinator.
 
 1. *Turn the display on*: Short press home (🏠).
 2. *Enable pairing mode*: Long press - (minus) for 3 seconds. WiFi logo is now blinking.
+
+### Broken OTA updates
+
+In the past there have been at least 2 revisions of this device where Zigbee2MQTT detected new firmwares that were incompatible with the device. See :
+* [BRT-100-TRV OTA broken / flash ZT5](https://github.com/Koenkk/zigbee2mqtt/issues/19173)
+* [Moes BRT-100-TRV not reporting after OTA update](https://github.com/Koenkk/zigbee2mqtt/issues/7941)
+
+In this situation a firmware update made the device unable to connect to the Zigbee network. As of Zigbee2MQTT 1.33.3 had been updated to disable OTA updates for the first revision with this problem but the last revision could still be updated with an incompatible firmware until the next release.
+
+As OTA updates are not under the control of Zigbee2MQTT which can only disable updates when problems occur. You should check that there isn't a faulty OTA update for your current device revision and avoid updating if there isn't a specific need for it.
```

Comment:
I don't understand this sentence, OTA updates are completely disabled, what is there to be checked?

Reply:
Without looking at the code history but only at the discussion in the Zigbee2MQTT #7941 issue I mistakenly understood that OTA updates were disabled in July of 2021 for the current revision but the problem came back for newer revisions in September.

I did further digging and found your fix (https://github.com/Koenkk/zigbee-herdsman-converters/commit/26951d7fba9636c34c4d7da09fc40de49f817981) so I understand that the last sentence isn't relevant.

I would just leave a comment about why OTA updates are forbidden to : 
- avoid wasting time explaining why to users,
- give an opportunity to people using old releases to find this information before they brick their device.

I have no idea if there are any people using old releases out there with these devices so my second point might not be relevant.

Reply:
I suggest to simplify this and line 41 to:

> In this situation a firmware update made the device unable to connect to the Zigbee network. As of Zigbee2MQTT 1.33.3 all OTA updates for this device have been disabled, if you are running an older Zigbee2MQTT version it is recommended to **not** do any OTA updates.

Reply:
Yes that's concise and clear. Do you want me to prepare a last pull request with it?

Reply:
You can update this one.

Reply:
Done (I think). I just learned that pushing new commits to a branch used for a GitHub pull request updates the pull request.
I'm used to git bare-bones, GitHub's interface for pull requests is new to me...